lib/webpack/util.js

```diff
@@ -0,0 +1,49 @@
+const { promisify } = require('util')
```

Comment:
You can use `fs-extra` here without having to `promisify`.

Reply:
Oh, didn't know it exists. I will do this now.